### PR TITLE
Inventory

### PR DESCRIPTION
**Pull request by charmhorse**

### CODE REVIEW

File: src/pages/inventory/transaction.jsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+            to="../"
+            className="absolute right-[10%] bottom-[7%] w-[150px] py-[10px] border-minionRed text-minionRed text-[20px] font-[900] border-[2px] text-center rounded-md"
+          >
+            Go Back
+          </Link>
+        </div>
+      </div>
+    </>
+  );
+}
```
Comment (pandablue0809) [inline src/pages/inventory/transaction.jsx:206]: plz remove unused const

File: src/pages/inventory/router.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect, memo } from "react";
```
Comment (dot425) [inline src/pages/inventory/router.jsx:1]: remove

Comment (smartdev0809) [reply src/pages/inventory/router.jsx:1]: remove

**Review by pandablue0809 — APPROVED** [review]

**Review by dot425 — APPROVED** [review]

**Review by smartdev0809 — APPROVED** [review]

**Comment by pandablue0809** [comment]
LGTM

**Comment by dot425** [comment]
LGTM

**Comment by smartdev0809** [comment]
LGTM

**Pull request merged**